CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via [issue on our bug tracker](https://trac.osgeo.org/osgeolive/report/1),
+[OSGeoLive mailing list](https://lists.osgeo.org/mailman/listinfo/osgeolive), or any other method with the owners of this repository before making a change. 
+
+By submitting you are agreeing to make your content available via our [open license policy](https://live.osgeo.org/en/copyright.html).
+
+In the same manner, when translating on Transifex, you are agreeing to make your translations following those licence terms.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```

Comment:
Missing link to code of conduct.

Reply:
Hmm, too many things in one file, (just saw the code of conduct down)

Reply:
> Missing link to code of conduct.

Shall it be this one? https://www.osgeo.org/code_of_conduct/

Reply:
yes

Reply:
I will close this one and merge. We can make improvement on another PR